odl/space/npy_tensors.py

```diff
@@ -1262,10 +1264,30 @@ def imag(self):
 
         Examples
         --------
+        Access the imaginary part of a complex vector:
+
         >>> space = odl.cn(3)
         >>> x = space.element([1 + 1j, 2, 3 - 3j])
         >>> x.imag
         rn(3).element([ 1.,  0., -3.])
+
+        Set the imaginary part:
+
+        >>> space = odl.cn(3)
+        >>> x = space.element([1 + 1j, 2, 3 - 3j])
+        >>> zero = odl.rn(3).zero()
+        >>> x.imag = zero
+        >>> x
+        cn(3).element([(1+0j), (2+0j), (3+0j)])
+
+        Other array-like types and broadcasting:
+
+        >>> x.imag = 1.0
+        >>> x
+        cn(3).element([(1+1j), (2+1j), (3+1j)])
+        >>> x.imag = [2, 3, 4]
+        >>> x
+        cn(3).element([(1+2j), (2+3j), (3+4j)])
```

Comment:
This example has the best text headers IMO, use this style also in the other places.